webview/src/experiments/components/table/styles.module.scss

```diff
@@ -661,7 +663,8 @@ $bullet-size: calc(var(--design-unit) * 4px);
 }
 
 .copyButton {
```

Comment:
The copy button isn't appearing when I'm hovering on cells: 

![image](https://user-images.githubusercontent.com/43496356/213747078-bc83cbe3-c685-41e8-82c1-1c03d41df759.png)

Reply:
It's one of the first thing I fixed, but I guess it got lost with the merge. Fixed back now.